history-of-quakers/modernized/04-chapter-2.adoc

```diff
@@ -714,7 +753,7 @@ Then he told him, he was; and so the man rode away to the justice.
 
 Now the men that guarded G. Fox, said, it would be well,
 if the justice was not drunk when they came to him, because he used to be drunk early,
-G+++.+++ Fox being brought in before him, and not putting off his hat, and saying you to him,
+G+++.+++ Fox being brought in before him, and not putting off his hat, and saying _you_ to him,
```

Comment:
ahhh, this looks like a place where jason's auto-modernization should have left it `thou` -- can you change it back?